### DISCUSSION TITLE
How to improve the accurate_ratio of the dpgen progress?

### DISCUSSION THREAD

**Discussion by Guancred** [deepmodeling/dpgen#1448] [Q&A]
Hi, all

I have trained the dp model by dpgen with 17 iteration, however, the accurate_ratio is only 46.9520%, as shown in the fig, 
so, whether there are some method for me to improve the efficiency?
Thanks very much! 
![image](https://github.com/deepmodeling/dpgen/assets/91767936/9c1708a7-ef95-4a45-b15c-c9c04489b942)

**Comment by KeouLee** [comment]
Hello, I've meet the same problem, that my accuracy ratio converges to ~88% after 18 iteration. Have you solved the problem yet?

**Reply by Guancred** [reply]
Sorry,  this problem still exists, and the accuracy ratio maintain 60% after 30 iteration, i also have no ideal for this..
